hack/go-lint.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+if [ "$IS_CONTAINER" != "" ]; then
+  golint -set_exit_status "${1:-.}"
```

Comment:
nit: I'd prefer `"${@}"` to `${1:-.}"` so we can pass multiple paths.  But as long as we're only linting `installer/...`, both approaches are fine.